build_tools/cmake/iree_run_module_test.cmake

```diff
@@ -0,0 +1,369 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Utility function to return the platform name in crosscompile.
+function(iree_get_platform PLATFORM)
+  if(ANDROID AND CMAKE_ANDROID_ARCH_ABI STREQUAL "arm64-v8a")
+    set(_PLATFORM "android-arm64-v8a")
+  elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64")
+    set(_PLATFORM "x86_64")
+  else()
+    set(_PLATFORM "${CMAKE_SYSTEM_PROCESSOR}-${CMAKE_SYSTEM_NAME}")
+  endif()
+  set(${PLATFORM} "${_PLATFORM}" PARENT_SCOPE)
+endfunction()
+
+# iree_run_module_test()
+#
+# Creates a test using iree-run-module to run an IREE module (vmfb).
+#
+# The function supports cross compile and checks the benchmark suite prefixes
+# based on the cross compile platform settings, so there is no matching bzl rule
+# until there is a bazel --build_config for cross compile targets in place.
```

Comment:
This doesn't do anything with weird benchmark suites and I don't think there's any reason it couldn't work in Bazel also. You could just say it's unimplemented in Bazel because it's not used there